ts/features/lollipop/screens/UnsupportedDeviceScreen.tsx

```diff
@@ -1,61 +1,35 @@
 import I18n from "i18n-js";
-import { Text as NBButtonText } from "native-base";
 import * as React from "react";
-import { View, SafeAreaView, StyleSheet, Modal } from "react-native";
-import { VSpacer, Pictogram } from "@pagopa/io-app-design-system";
-import ButtonDefaultOpacity from "../../../components/ButtonDefaultOpacity";
-import { Body } from "../../../components/core/typography/Body";
-import { H3 } from "../../../components/core/typography/H3";
-import { IOStyles } from "../../../components/core/variables/IOStyles";
+import { Modal } from "react-native";
 import { unsupportedDeviceLearnMoreUrl } from "../../../config";
-import themeVariables from "../../../theme/variables";
 import { openWebUrl } from "../../../utils/url";
 import { useAvoidHardwareBackButton } from "../../../utils/useAvoidHardwareBackButton";
-
-const styles = StyleSheet.create({
-  errorContainer: {
-    flex: 1,
-    justifyContent: "center",
-    alignItems: "center",
-    padding: 56
-  },
-  buttonContainer: {
-    justifyContent: "center",
-    alignItems: "center",
-    padding: themeVariables.contentPadding
-  },
-  title: {
-    textAlign: "center"
-  }
-});
+import { OperationResultScreenContent } from "../../../components/screens/OperationResultScreenContent";
 
 const handleLearnMorePress = () => {
   openWebUrl(unsupportedDeviceLearnMoreUrl);
 };
 
-// This component doesn't need a BaseHeaderComponent.
-// It Represents a blocking error screen that you can only escape with the rendered button(s).
-// A new template is coming soon: https://pagopa.atlassian.net/browse/IOAPPFD0-71
+// This component Represents a blocking error screen that you can only escape with the rendered button(s).
 const UnsupportedDeviceScreen = () => {
   useAvoidHardwareBackButton();
+
+  const title = I18n.t("unsupportedDevice.title");
+  const subtitle = I18n.t("unsupportedDevice.subtitle");
+  const secondaryAction = {
```

Comment:
@shadowsheep1 it can be optimized with useMemo, but i need to understand if it re-renders in response to localization updates

Reply:
You can for sure use `useMemo` if you like, because both device and profile locale changes need a fresh start of the app. The former is made by the OS, the latter is asked to the citizen.